.circleci/src/workflows/commit.yml

```diff
@@ -5,39 +5,108 @@ jobs:
         - mattermost
       delivery: none
 
+  - build_web:
```

Comment:
Would it maybe make sense to split the commit-workflow into to workflows (one for feautre branches and one for main) as they share almost nothing and we can get rid of the multiple filter-checks?

Reply:
Would be nice, yes. But AFAIK it is not possible to filter workflows by branch name. So it would be possible to split it in two files but we'd still need the filters :/

Reply:
Makes sense to split anyway though, you are right :+1:

Reply:
Maybe it is possible after all, will try :thinking:

Reply:
Seems to work, thanks for asking!